src/services/mondomarket/index.jsx

```
@@ -0,0 +1,473 @@
+                spinner={isLoading ? <Spinner size="sm" /> : null}
+              />
+            </div>
+          </div>
+        )}
+      </Form>
+    </>
+  );
+};
+
+export const MondoDetailsViewer = ({
+  mondo,
```

ESLint

[error] 'mondo' is missing in props validation

(react/prop-types)